autogpt_platform/frontend/src/lib/supabase/server/getServerSupabase.ts

```diff
@@ -18,7 +18,12 @@ export async function getServerSupabase() {
           setAll(cookiesToSet) {
             try {
               cookiesToSet.forEach(({ name, value, options }) =>
-                cookieStore.set(name, value, options),
+                cookieStore.set(name, value, {
+                  ...options,
+                  httpOnly: true,
+                  secure: process.env.NODE_ENV === "production",
+                  sameSite: "lax",
+                }),
```

Comment:
Why are we not using "strict"

nit: We can make a helper out of this

Reply:
https://stackoverflow.com/questions/59990864/what-is-the-difference-between-samesite-lax-and-samesite-strict

I think we want lax still but lmk

biggest thing is the second answer which lax will help but strict will be werid on

Reply:
this also has a lot of clarity but i think lax is still good https://web.dev/articles/samesite-cookies-explained